app/assets/stylesheets/components/_card.scss

```diff
@@ -3,90 +3,168 @@
-      background-color: nhsuk-colour("red");
+    background-color: nhsuk-tint(nhsuk-colour("red"), 80%);
+    border-color: nhsuk-tint(nhsuk-colour("red"), 50%);
+    color: nhsuk-shade(nhsuk-colour("red"), 50%);
+
+    &.nhsuk-card__clickable:active {
+      border-color: nhsuk-shade(nhsuk-colour("red"), 50%);
     }
   }
 
-  &--reversed {
-    background-color: nhsuk-colour("blue");
-    border-color: nhsuk-shade(nhsuk-colour("blue"), 25%);
-    color: nhsuk-colour("white");
+  &--yellow {
+    background-color: nhsuk-tint(nhsuk-colour("warm-yellow"), 80%);
+    border-color: nhsuk-colour("warm-yellow");
 
-    &:active {
-      border-color: nhsuk-shade(nhsuk-colour("blue"), 50%);
+    .nhsuk-card__heading--feature {
+      background-color: nhsuk-colour("warm-yellow");
+      color: $nhsuk-text-colour;
+    }
+
+    blockquote {
+      border-color: nhsuk-colour("warm-yellow");
     }
+  }
 
-    .nhsuk-card__link {
-      @include nhsuk-link-style-white;
+  &--aqua-green,
+  &--blue,
+  &--green,
+  &--grey,
+  &--red,
+  &--yellow {
+    .nhsuk-card__link,
+    .nhsuk-card__link:hover {
+      color: inherit;
     }
   }
 
   &--offset {
     background-color: rgba(nhsuk-colour("grey-5"), 0.5);
   }
+}
 
-  &--data {
-    display: flex;
+.app-card--compact {
+  @include nhsuk-responsive-margin(3, "bottom");
 
-    .nhsuk-card__content {
-      display: flex;
-      flex-direction: column;
-      flex-grow: 1;
-    }
+  .nhsuk-card__heading {
+    @include nhsuk-responsive-margin(1, "bottom");
+  }
 
-    .nhsuk-card__heading {
-      @include nhsuk-typography-weight-normal;
-    }
+  .nhsuk-card__content {
+    @include nhsuk-responsive-padding(4);
+    @include nhsuk-responsive-padding(3, "bottom");
+  }
+}
 
-    .nhsuk-card__description {
-      margin-top: auto;
+.app-card--dense {
+  @include nhsuk-responsive-margin(2, "bottom", $important: true);
 
-      @include nhsuk-font-size(48);
-      @include nhsuk-typography-weight-bold;
-    }
+  .nhsuk-card__heading {
+    @include nhsuk-responsive-margin(1, "bottom");
+  }
+
+  .nhsuk-card__content {
+    @include nhsuk-responsive-padding(3);
+    @include nhsuk-responsive-padding(2, "bottom");
   }
 }
 
-.app-card--compact {
-  @include nhsuk-responsive-margin(3, "bottom");
+// Align counts across cards
+.nhsuk-card:has(> .nhsuk-card__content > .app-card__count) {
+  display: flex;
 
   .nhsuk-card__heading {
-    @include nhsuk-responsive-margin(1, "bottom");
+    line-height: 1.2;
   }
 
   .nhsuk-card__content {
-    @include nhsuk-responsive-padding(4);
-    @include nhsuk-responsive-padding(3, "bottom");
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+}
+
+.app-card__count {
+  @include nhsuk-font-size(48);
+  @include nhsuk-typography-weight-bold;
+
+  .app-card--dense & {
+    @include nhsuk-font-size(26);
+  }
+}
+
+.app-card__heading--xxs {
+  @include nhsuk-font(16, $weight: bold);
+}
+
+.app-card__heading {
```

Comment:
These changes will affect other cards… I thought I’d already made these changes, but evidently not. Probably worth me creating a separate PR to handle how we handle styling featured card headings versus entire cards.

Reply:
Addressed in https://github.com/nhsuk/manage-vaccinations-in-schools/pull/4671